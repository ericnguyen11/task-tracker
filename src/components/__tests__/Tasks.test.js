import { MemoryRouter } from "react-router-dom";

import { shallow } from "enzyme";

import Tasks from "../Tasks.js";

it("renders without crashing", () => {
  shallow(
    <MemoryRouter>
      <Tasks />
    </MemoryRouter>
  );
});

// Providing testKey guarantees snapshot doesn't change from random generated key
it("matches snapshot", () => {
  const wrapper = shallow(
    <MemoryRouter initialEntries={[{ pathname: "/", key: "testKey" }]}>
      <Tasks />
    </MemoryRouter>
  );
  expect(wrapper).toMatchSnapshot();
});
