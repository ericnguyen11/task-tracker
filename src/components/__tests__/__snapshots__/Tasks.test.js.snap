// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<Router
  location={
    Object {
      "hash": "",
      "key": "testKey",
      "pathname": "/",
      "search": "",
      "state": null,
    }
  }
  navigationType="POP"
  navigator={
    Object {
      "action": "POP",
      "back": [Function],
      "block": [Function],
      "createHref": [Function],
      "forward": [Function],
      "go": [Function],
      "index": 0,
      "listen": [Function],
      "location": Object {
        "hash": "",
        "key": "testKey",
        "pathname": "/",
        "search": "",
        "state": null,
      },
      "push": [Function],
      "replace": [Function],
    }
  }
>
  <Tasks />
</Router>
`;
